spec/support/fake_bisect_runner.rb

```diff
@@ -1,3 +1,5 @@
+require 'set'
```

Comment:
Sure, makes total sense. Will reimplement with arrays, I think (unless you're happy for me to start adding methods like `#superset?` to the substitute `Set` implementation...)